.github/workflows/pytest-core-nompi.yml

```diff
@@ -86,12 +86,12 @@ jobs:
           language: "C"
           sympy: 1.8
 
-        - name: pytest-docker-py36-gcc-omp
-          python-version: 3.6
-          os: ubuntu-18.04
+        - name: pytest-docker-py310-gcc-omp
```

Comment:
I wouldn't change the docker build we wanna keep a safe standard version for that one.